container_files/assets/www/doc/builtin/functions/Stemmer.md

```diff
@@ -79,11 +102,11 @@ PUT /v1/functions/my_stemmer
 
 we can apply the stemming algorithm on the provided string:
 
-`SELECT my_stemmer({document: 'I like having lots'})`
+`SELECT my_stemmer({document: 'I like having celeries'})`
```

Comment:
amusingly, neither celeries nor celeri is a word in english... we should put carrots